### ISSUE TITLE
MP4 file are not loaded due to CORS error

### ISSUE THREAD

**Issue by abizeauMaxon** [Adobe-CEP/CEP-Resources#425]
Our panel is currently failing to load some mp4 (try with other formats even gif) and still having CORS error as follow:

> Access to XMLHttpRequest at 'file://localhost/.../plugin-preview.mp4' from origin 'file://' has been blocked by CORS policy: Cross origin requests are only supported for protocol schemes: http, data, chrome, chrome-extension, chrome-untrusted, https.

The manifest has the following paramater:

```
<CEFCommandLine>                       
    <Parameter>--enable-nodejs</Parameter>                   
    <Parameter>--disable-accelerated-video-decode</Parameter>                     
</CEFCommandLine>
```

And I recently add those but nothing change

```
    <Parameter>--allow-file-access</Parameter>
    <Parameter>--allow-file-access-from-files</Parameter>       
```

I really don't understand what is going on. Note that our panel was set to work with CEP 5.0 and I move up to 9.0/10.0/11.0 inside After Effects 2022 and none of those seems to do a change.

**Comment by ErinFinnegan** [comment]
Did you read the other issues about CORS errors?

https://github.com/Adobe-CEP/CEP-Resources/issues/365
https://github.com/Adobe-CEP/CEP-Resources/issues/364

**Comment by abizeauMaxon** [comment]
> Did you read the other issues about CORS errors?
> 
> #365 #364

Yes I did. And I'm not using iframe, directly using XMLHttpRequest to load some file on disk.

**Comment by ErinFinnegan** [comment]
Ah, well, looking through my backlog here's what I've got:

> With CEP 11, with security updates, we are seeing that a lot more requests are being blocked by CORS checks. The extension may encounter this if it is trying to access an endpoint that does not have the Access-Control-Allow-Origin header. Please ensure that the endpoints that are being accessed have Access-Control-Allow-Origin headers set appropriately
